### PR TITLE
SC-5560 - provide-click-dummy

### PR DESCRIPTION
**Pull request by ABialoskorski**
# Description

## Links to Tickets or other pull requests

- https://ticketsystem.schul-cloud.org/browse/SC-5560

## Screenshots of UI changes
image
localhost_3100_(iPhone 6_7_8) (1)

### CODE REVIEW

File: views/authentication/home.hbs [hunk 1]
```diff
@@ -27,8 +27,45 @@
...
+                                    Demozugang anfordern
+                                </a>
+                            </div>
```
Comment (BlackGhostHD) [inline views/authentication/home.hbs:61]: This is the new part for Ghost, right?

Comment (ABialoskorski) [reply views/authentication/home.hbs:61]: yes, only that div

**Comment by ABialoskorski** [comment]
merge only css, html div needs to be added in ghost

**Comment by ABialoskorski** [comment]
https://stackoverflow.com/questions/1472506/broken-accented-characters-in-a-mailto-link

**Pull request merged**